implementations/rust/ockam/ockam_command/src/status/static/long_about.txt

```diff
@@ -0,0 +1,10 @@
+The `status` command displays the current status of your machine. This includes information such as
```

Comment:
> The `status` command displays the current status of your machine.

I'd delete this sentence and start the next one with `This commands shows information such as...`